### ISSUE TITLE
[Suggestion]:  Could you add an option to disable the mod's mob death animations since they overlap with the Physics Mod animations, since there is an incompatibility in some ways?

### ISSUE THREAD

**Issue by Mase100x** [Yesssssman/epicfightmod#1587]
### Have you checked if there's any similar suggestions ?

- [X] I checked, there are no similar suggestions that have been reported.

### Are you using the latest Epic Fight ?

- [X] I checked I'm using latest Epic Fight version.

### Is this suggestion related to another mod? (i.e: add animations for X mod)

- [X] This is related to other mods.

### The mod

Physics Mod :  https://www.curseforge.com/minecraft/mc-mods/physics-mod

### Minecraft Version

1.20.1

### What's your suggestion?

In this case I would like there to be this option to be able to enjoy the animations of the Physics Mod when eliminating them, since with this mod you can enjoy more since with the "Rag Doll" animation you can have a great experience when eliminating some mob, it would only be disabling said death animation, this mod is incredible and I love it and I know the effort that you as developers put into this mod and I think that this suggestion could contribute more to the mod, Thank you very much for your attention, happy day .

**Comment by LandaAzul** [comment]
I agree with him, some option to disable death animations would be great.

**Comment by guivnf** [comment]
That's basically impossible in the way we have it coded right now. It'd take months to fully rewrite the code logic behind it, and so unfortuntately i don't think we'll ever have a way to straight up disable them. What we do want to do is allow users to add more death variations and maybe in the future add physics based deaths, with our own rag dolls (maybe even supporting custom modded entities from other mods, which physics mod doesn't handle)

**Comment by Dickenbauss** [comment]
That would be amazing, ive always been really disappointed with how the physics mod cant even support textured existing entities much less custom modded entities.

Ive seen other people mention this on reddit too. Would be great if you added ragdolls, may i ask if you plan on adding them across all versions, or just the most recent ones? 

Thanks for the great mod btw
